homeassistant/components/upnp/__init__.py

```diff
@@ -222,13 +234,28 @@ class UpnpEntity(CoordinatorEntity):
 
     coordinator: UpnpDataUpdateCoordinator
 
-    def __init__(self, coordinator: UpnpDataUpdateCoordinator) -> None:
+    def __init__(
+        self,
+        coordinator: UpnpDataUpdateCoordinator,
+        entity_description: UpnpSensorEntityDescription,
+    ) -> None:
         """Initialize the base entities."""
         super().__init__(coordinator)
         self._device = coordinator.device
+        self.entity_description: UpnpSensorEntityDescription = entity_description
```

Comment:
This typing needs to move to under line 235

Reply:
Ah, now I understand what you meant with your other comment about "Add this under here". :-)